src/graph_notebook/notebooks/01-Getting-Started/01-About-the-Neptune-Notebook.ipynb

```diff
@@ -269,14 +373,21 @@
     "\n",
     "[Explore English Premier League using SPARQL](../02-Visualization/EPL-SPARQL.ipynb)\n",
     "\n",
+    "[Explore English Premier League using openCypher](../02-Visualization/EPL-openCypher.ipynb)\n",
+    "\n",
     "Curious about the business problems can be solved with graph? Check out these sample application notebooks for some inspiration.\n",
     "\n",
     "[Introduction to Fraud Graphs](../03-Sample-Applications/01-Fraud-Graphs/01-Building-a-Fraud-Graph-Application.ipynb)\n",
     "\n",
-    "[Introduction to Knowledge Graphs](../03-Sample-Applications/02-Knowledge-Graphs/01-Building-a-Knowledge-Graph-Application.ipynb)\n",
+    "[Introduction to Knowledge Graphs](../03-Sample-Applications/02-Knowledge-Graphs/01-Building-a-Knowledge-Graph-Application-Gremlin.ipynb)\n",
```

Comment:
Suggesting this link instead, which lets user choose whether they want the Gremlin or OC version: https://github.com/aws/graph-notebook/tree/main/src/graph_notebook/notebooks/03-Sample-Applications/02-Knowledge-Graphs

Reply:
Done, using the base folder link.